packages/render-fidelity-tools/test/config.json

```diff
@@ -75,6 +75,23 @@
         "radius": 11.2
       }
     },
+    {
+      "name": "khronos-AttenuationTest",
+      "model": "../../../shared-assets/models/glTF-Sample-Models/2.0/AttenuationTest/glTF/AttenuationTest.gltf",
+      "dimensions": {
+        "width": 512,
+        "height": 1024
+      },
+      "target": {
+        "y": 3.6
+      },
+      "orbit": {
+        "radius": 11.2
```

Comment:
I would start with the default values (just removing these listings), then add back in the ones you need until the model is nicely framed. Only use custom dimensions if the model has a really odd aspect ratio.

Reply:
Sorry I just saw this. Let me update them in the next commit

Reply:
Just sent another commit let me know what you think